Mage/src/main/java/mage/abilities/effects/common/ReturnFromExileForSourceEffect.java

```diff
@@ -84,25 +90,34 @@ public ReturnFromExileForSourceEffect withText(boolean pluralCards, boolean plur
         return this;
     }
 
+    public ReturnFromExileForSourceEffect withPutVerb() {
```

Comment:
I know false is the default, but generally the style I've seen is to take a boolean parameter